features/bootstrap/SpeakerContext.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+use Behat\MinkExtension\Context\RawMinkContext;
+use App\DataFixtures\SpeakerFixtures;
+
+final class SpeakerContext extends RawMinkContext
+{
+    private const PATH_SPEAKER = '/speakers/';
+    
+    /**
+     * @When /^I am on the speaker listing page$/
+     */
+    public function iAmOnTheSpaekerListingPage(): void
```

Comment:
```suggestion
    public function iAmOnTheSpeakerListingPage(): void
```

Reply:
oui ?

Reply:
What didn't you understand about my suggestion?